Documentation/rust/quick-start.rst

```diff
@@ -56,16 +58,16 @@ If ``rustup`` is being used, run::
 The components are installed per toolchain, thus upgrading the Rust compiler
 version later on requires re-adding the component.
 
-Otherwise, if a standalone installer is used, the Rust repository may be cloned
-into the installation folder of the toolchain::
+Otherwise, if a standalone installer is used, the Rust source tree may be
+downloaded into the toolchain's installation folder::
 
-	git clone --recurse-submodules \
-		--branch $(scripts/min-tool-version.sh rustc) \
-		https://github.com/rust-lang/rust \
-		$(rustc --print sysroot)/lib/rustlib/src/rust
+	curl -L https://static.rust-lang.org/dist/rust-src-$(scripts/min-tool-version.sh rustc).tar.gz | \
```

Comment:
Is `-L` needed? Should we have other options, like `--proto '=https' --tlsv1.2 -sSf`?

Also, is the `\` at the end needed?

Reply:
Without `-L` it would break if this url changes into a redirect in the future. The `\` at the end is necessary to prevent a shell from trying to run the partial command at the newline rather than interpreting everything as a single command.

Reply:
> Without `-L` it would break if this url changes into a redirect in the future

Yeah, I meant whether there was a redirect going on now already.

I guess the `-L` won't hurt, but hopefully the URL won't change.

> The `\` at the end is necessary to prevent a shell from trying to run the partial command at the newline rather than interpreting everything as a single command.

The `|` already does that, no? (even in the interactive case)

Reply:
Yeah, in my case I used to put the `|` in the next line, so that it looked like what one does in some functional languages. But since the `\` could be avoided, I started putting it at the end.

Reply:
I have updated this to drop the `\` but kept the `-L`, just in case